scripts/in_container/bin/run_tmux

```diff
@@ -55,16 +55,21 @@ tmux send-keys -t 'Main' 'bash' C-m 'clear' C-m
 
 tmux split-window -v
 tmux select-pane -t 1
-tmux send-keys 'opentelemetry-instrument airflow scheduler' C-m
+tmux send-keys '${COMMAND_PREFIX} airflow scheduler' C-m
 
 tmux split-window -h
 tmux select-pane -t 2
-tmux send-keys 'opentelemetry-instrument airflow webserver' C-m
+tmux send-keys '${COMMAND_PREFIX} airflow webserver' C-m
 
 if python -c 'import sys; sys.exit(sys.version_info < (3, 7))'; then
     tmux select-pane -t 0
     tmux split-window -h
-    tmux send-keys 'opentelemetry-instrument airflow triggerer' C-m
+    tmux send-keys '${COMMAND_PREFIX} airflow triggerer' C-m
+fi
+
+COMMAND_PREFIX=""
```

Comment:
This one should be moved before the tmux commands - otherwise it will not be used.

Reply:
Do you mean I should move the command prefix above?